scripts/requirements.txt

```diff
@@ -18,3 +18,4 @@ windows-curses; sys_platform == "win32"
 colorama
 Pillow
 intelhex
+cmake==3.13.1
```

Comment:
I agree with @carlescufi. Better to just keep telling people to install it (as we already do in the doc) that to add it here.
If I were a new user, say in 1 year, and I already have a newer version than this installed, it would be pretty annoying to have this one pulled thru pip.
& the fixed version makes it seem that things won't work with newer versions.